CITATION.rst

```diff
@@ -1 +1 @@
-sunpy/CITATION.rst
\ No newline at end of file
+sunpy/CITATION.rst
```

Comment:
I still think there should be no difference in this file. It is a symlink in the repo and I am weary of a change here.

Reply:
Ya I know, i just added whats there in the master branch in sunpy

Reply:
There should be no changes to this file in your pull request. It should be the same as the code in master.

Reply:
I rebased the whole thing, created a new branch with resetting commits prior to editing this file and i have merged into m11dedhia/sunpy master repository. Will this do?

Reply:
No since the file has still changed in this PR.

Reply:
Closing and opening a new PR won't change anything.

If you follow: https://stackoverflow.com/questions/1817766/how-to-revert-to-origins-master-branchs-version-of-file you should be able to reset this file.